src/mail/view/ConversationViewer.ts

```diff
@@ -142,27 +142,30 @@ export class ConversationViewer implements Component<ConversationViewerAttrs> {
 
 	private renderViewer(mailViewModel: MailViewerViewModel, isPrimary: boolean, position: number | null): Children {
 		return m(
-			".border-radius-big.rel",
-			{
-				class: mailViewerMargin(),
-				key: elementIdPart(mailViewModel.mail.conversationEntry),
-				style: {
-					border: `1px solid ${theme.list_border}`,
-					backgroundColor: theme.content_bg,
-					marginTop: px(conversationCardMargin),
+			".mlr-safe-inset",
```

Comment:
why re-wrap it and not add it to the existing element one line below?

Reply:
Ah, yeah I tried that at first but it clashed with the mailViewerMargin class that gets added on that one, thus the wrap. Would there be a better way to handle that?

Reply:
ah I didn't notice it! I think it's fine as is then